.github/actions/chromedriver-updater/chromedriver_updater/constants.py

```diff
@@ -0,0 +1,33 @@
+#  Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
The only thing stopping this file from getting a perfect linting score is the lack of a module docstring, so that would be an easy win.